source/photo.html

```diff
@@ -40,8 +47,13 @@ <h1 class="page-main__title page-main__title--inner">Фото и видео</h1>
       <section class="page-main__photo-gallery photo-gallery">
         <h2 class="visually-hidden">Фотогалерея</h2>
         <article class="photo-gallery__article photo-gallery__article--first photo">
-          <a class="photo__link" href="#">
-            <img src="img/photo-1-mobile.jpg" width="280" height="280" alt="Фото загородной автодороги">
+          <a class="photo__link photo__link--first" href="#">
+            <picture>
+              <source media="(min-width: 1200px)" srcset="img/photo-1--desktop@1x.jpg 1x, img/photo-1--desktop@2x.jpg 2x, img/photo-1--desktop@3x.jpg 3x">
+              <source media="(min-width: 768px)" srcset="img/photo-1--tablet@1x.jpg 1x, img/photo-1--tablet@2x.jpg 2x, img/photo-1--tablet@3x.jpg 3x">
+              <source media="(min-width: 320px)" srcset="img/photo-1--mobile@2x.jpg 2x, img/photo-1--mobile@3x.jpg 3x">
+              <img src="img/photo-1--mobile@1x.jpg" width="280" height="280" alt="Фото загородной автодороги">
```

Comment:
А почему не так?
```suggestion
              <img src="img/photo-1--mobile@1x.jpg" srcset="img/photo-1--mobile@2x.jpg 2x, img/photo-1--mobile@3x.jpg 3x" width="280" height="280" alt="Фото загородной автодороги">
```

И для остальных изображений в галерее тот же вопрос.

Reply:
Так - просто так, для разнообразия.